amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpConnectionProvider.scala

```diff
@@ -203,6 +203,9 @@ object AmqpDetailsConnectionProvider {
   def apply(host: String, port: Int): AmqpDetailsConnectionProvider =
     new AmqpDetailsConnectionProvider(immutable.Seq(host -> port))
 
+  def apply(hostAndPorts: immutable.Seq[(String, Int)]): AmqpDetailsConnectionProvider =
```

Comment:
Please add it to the Java API with `j.u.Collection<akka.japi.Pair<String, Integer>>`, as well.

Reply:
Done

Reply:
You need to add some conversion logic...

Reply:
Oh, I forgot, sorry about this. Fixed.

Reply:
Oops, your change is adapting the wrong data.

Reply:
I'm Junior Backend developer :)